apiary-metastore-events/kafka-metastore-events/kafka-events-common/src/test/java/com.expediagroup.apiary.extensions.events.metastore/common/listener/AbstractMetaStoreEventListenerTest.java

```diff
@@ -349,4 +346,11 @@ public void onDropFunction() throws Exception {
     verify(eventSerDe, never()).marshal(any(SerializableListenerEvent.class));
   }
 
+  private void verifyMetrics(String metricName) {
```

Comment:
Is it worth testing that no other metrics have been sent? Uncertain how easy this is, can you just ask for all counters and ensure there aren't any others?

Reply:
Well there are 40+ other jvm metrics being reported, but this is the only counter, so can check for that at least